src/main/java/com/EstimatedDeliveryDate/Entity/CutOffTime.java

```diff
@@ -0,0 +1,31 @@
+package com.EstimatedDeliveryDate.Entity;
+
+import java.io.Serializable;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBDocument;
+
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/*Document object for the CutOffTimeList*/
+@Setter
+@NoArgsConstructor
+@DynamoDBDocument
+public class CutOffTime implements Serializable{
+	private static final long serialVersionUID = 1L;
+	private String cutoffTime;
```

Comment:
Are we planning to change this to a time object in business logic, or in DAO itself?

Reply:
Yes.we are converting it in business logic.